packages/components/src/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { BADGE_SIZE_VALUES } from './Badge.types';
+import Badge from './Badge';
+
+const meta: Meta<typeof Badge> = {
+    component: Badge,
+    parameters: {
+        layout: 'centered',
+    },
+    tags: ['autodocs'],
+    argTypes: {
+        value: {
```

Comment:
Do you need this property here? Most of argTypes are correctly identified by storybook (className isn't, that's why it's needed ;)) but as you use value arg I guess it should be properly recognized automatically